apps/core/scenes/search/PickersWrapper.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { LAYOUT } from '@kiwicom/margarita-utils';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+type Props = {|
+  +children: React.Node,
+  +layout: number,
+|};
+
+export default function PickersWrapper({ children, layout }: Props) {
+  const desktopLayout = layout >= LAYOUT.desktop;
+  const rowLayout = layout >= LAYOUT.largeMobile;
+
+  return (
+    <View
+      style={[
+        styles.container,
+        desktopLayout && styles.desktopContainer,
+        rowLayout && styles.rowContainer,
```

Comment:
Should we think about some more reuasable way of handeling this?

Reply:
I mean some kind of `media-query` wrapper [react-native-web-docs](https://github.com/necolas/react-native-web/blob/master/docs/guides/style.md#what-about-media-queries)

Reply:
@tbergq I've been thinking about something like
`<Media query={{ minWidth: LAYOUT.desktop }} style={styles.desktopContainer}>`
I'll try to review it and turn it into something more reusable. But this PR is starting to be quite big, so if you don't mind I'll create new issue for it.

Reply:
Yes that makes sense. New PR, and we can come back and improve this